Formula/goredo.rb

```diff
@@ -0,0 +1,27 @@
+class Goredo < Formula
+  desc "Go implementation of djb's redo, a Makefile replacement that sucks less"
+  homepage "http://www.goredo.cypherpunks.ru/"
+  url "http://www.goredo.cypherpunks.ru/download/goredo-0.11.0.tar.zst"
+  version "0.11.0"
+  sha256 "4608b75bc8a3dfb62ef305b5eed0f4b28ad39f64ae46baceea8100d90a8666f9"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+  depends_on "zstd" => :build
+
+  def install
+    goredo_prefix = "goredo-#{version}"
+    system "tar", "--use-compress-program", "unzstd", "-xvf", "#{goredo_prefix}.tar.zst"
+    cd goredo_prefix do
+      ENV["GOPATH"] = Pathname.pwd
+      system "go", "build", *std_go_args, "go.cypherpunks.ru/goredo"
+    end
+    cd bin do
+      system "./goredo", "-symlinks"
```

Comment:
Does this mean there is a definition of what to symlink defined in bin?

Reply:
I'm not sure what you're are saying, it basically a dist helper function to create the various redo-* symlinks to the binary.

Reply:
Where does the definition about those symlinks live?

Reply:
http://www.git.cypherpunks.ru/?p=goredo.git;a=blob;f=main.go;h=ed82856be37f842e6c66f9a0607d191388cb40ac;hb=HEAD#l76

I guess I could set them up explicitly manually in the package. Wdyt?

Reply:
I was expecting there to be some definition file. But if it's hardcoded I guess that's fine.

Reply:
So no changes required? I'm not sure what the next steps are. Do I mark the conversations are resolved? Not sure what to make of the "Resolve conversation" button.